conn.go

```diff
@@ -36,6 +36,8 @@ var (
 	errEmptyQueryID = errors.New("query id is empty")
 )
 
+var defaultKillQueryDeadline = time.Duration(time.Second)
```

Comment:
lets rename to defaultKillQueryTimeout because its actually a timeout, not a deadline